roles/principal_technical_consultant.md

```diff
@@ -0,0 +1,100 @@
+# Principal Technical Consultant
+
+Our Principal Technical Consultants define and drive strategy to deliver digital and technology outcomes for our customers. Working with senior customer stakeholders our consultants drive digital transformation.
```

Comment:
I'm unsure about the digital transformation angle we've taken here and further down the document. Feel too broad in scope and should limit around technology strategy (cloud, devops, software engineering, agile etc)